wiki.ros.org/metapackage_status.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+
+"""
+Generates wiki formatted output, which summarizes the status of a release's variants
+using the previous release's distribution cache and accounting for changes.
+
+To use it, checkout the <distro>-devel branch of the metapackages repo at
+
+https://github.com/ros/metapackages
+
+And then update the `path` variable near the top of the script to point to the directory
+containing the metapackages repo and run this script.
+"""
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from catkin_pkg.packages import find_packages
+from catkin_pkg.package import InvalidPackage
+from catkin_pkg.package import parse_package_string
+
+from rosdistro import get_cached_distribution
+from rosdistro import get_distribution_file
+from rosdistro import get_index
+from rosdistro import get_index_url
+from rosdistro.dependency_walker import DependencyWalker
+import argparse
+
+parser = argparse.ArgumentParser(
+    description='Generate wiki formatted output for the Maintenance status page.')
+parser.add_argument(
+    'current_distro', metavar='current_distro', type=str, nargs='?',
+    help='The distribution to generate a Wiki page for.', default='kinetic')
+parser.add_argument(
+    'prev_distro', metavar='prev_distro', type=str, nargs='?',
+    help='The distribution to generate a Wiki page for.', default='indigo')
```

Comment:
Does `indigo` as a default value make sense if the current distro defaults to `kinetic`?